src/components/ha-selector/ha-selector-object.ts

```diff
@@ -36,7 +41,20 @@ export class HaObjectSelector extends LitElement {
         : ""} `;
   }
 
+  protected updated(changedProps: PropertyValues) {
+    super.updated(changedProps);
+    if (
+      changedProps.has("value") &&
+      changedProps.get("value") &&
```

Comment:
Why `get(value)`?

I think we could be transitioning from undefined to a valid value, wouldn't we want to push the value in that case?

Reply:
yeah, it was to prevent `firstUpdated` in `ha-yaml-editor` and this to do the same, and call `setValue` twice.